### PR TITLE
Add CEKs (like jce-unlimited-strength-policy8) for Zulu casks

### PR DESCRIPTION
**Pull request by dpoggi**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-versions/pulls
[closed issues]: https://github.com/caskroom/homebrew-versions/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
Do you use both?

File: Casks/zulu8-cek.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    system_command '/bin/rm',
+                   args: ['-f', "/Library/Java/JavaVirtualMachines/zulu#{version}.jdk/Contents/Home/jre/lib/security/US_export_policy.jar"],
+                   sudo: true
+    system_command '/bin/cp',
+                   args: ['-an', "/Library/Java/JavaVirtualMachines/zulu#{version}.jdk/Contents/Home/jre/lib/security/local_policy.jar.bak", "/Library/Java/JavaVirtualMachines/zulu#{version}.jdk/Contents/Home/jre/lib/security/local_policy.jar"],
+                   sudo: true
+    system_command '/bin/cp',
+                   args: ['-an', "/Library/Java/JavaVirtualMachines/zulu#{version}.jdk/Contents/Home/jre/lib/security/US_export_policy.jar.bak", "/Library/Java/JavaVirtualMachines/zulu#{version}.jdk/Contents/Home/jre/lib/security/US_export_policy.jar"],
+                   sudo: true
+    system_command '/bin/rm',
```
Comment (commitay) [inline Casks/zulu8-cek.rb]: `cp` & `rm` could be replaced with `mv`

**Comment by dpoggi** [comment]
Azul is the last place I know of to get a recent build of Java 7 for Mac. I'm confident enough someone out there runs a proprietary Swing app that dies on 8 for whatever silly reason.

**Comment by commitay** [comment]
>  I'm confident enough someone out there runs a proprietary Swing app that dies on 8 for whatever silly reason.

I'd rather not add Casks just because someone might use them.

> caskroom/versions is not intended to be used for all and any old versions you personally require for xyz project; casks submitted here should be expected to be used by a reasonable number of people and supported by contributors long-term.

**Comment by dpoggi** [comment]
I do use both of these.

**Comment by commitay** [comment]
I don't think these or https://github.com/caskroom/homebrew-versions/pull/4539 should be in `versions`.

No one had a need for either of them when zulu 8 was in the main repo.

> casks submitted here should be expected to be used by a reasonable number of people

Ping @vitorgalvao

**Comment by dpoggi** [comment]
And I'd imagine "no one" ever installed them by hand. It seems a bit odd to not bat an eye at `jce-unlimited-strength-policy8` for the commercial JDK builds, then refuse inclusion of the identical component for their FOSS counterparts.

**Comment by commitay** [comment]
`jce-unlimited-strength-policy8` is an import from the main repo.

> It seems a bit odd to not bat an eye at jce-unlimited-strength-policy8 for the commercial JDK builds, then refuse inclusion of the identical component for their FOSS counterparts.

I'm not refusing them because of commercial vs FOSS, I don't think they meet the requirement for this repo that they be used by a reasonable number of people.

`zulu` 8 was in the main repo for 3 years and no one needed them.

**Comment by dpoggi** [comment]
Why does there seem to be such a strong preference for imports from the main repo? Does a cask having lived there for a while and gotten old enough to be versioned out somehow make it a better candidate for inclusion here?

Trying to understand the reasoning. I could see that preference coming from a desire for more eyeballs on code (more traffic to the main repo / latest versions) before it makes it here.

**Comment by commitay** [comment]
Please read the rules for acceptable Casks this repo.  https://github.com/caskroom/homebrew-versions#acceptable-casks

**Comment by dpoggi** [comment]
¯\_(ツ)_/¯ seems rigid, but I'm not the one here with a sense for the maintenance burden all this entails. Closing these PRs.

**Comment by vitorgalvao** [comment]
> seems rigid, but I'm not the one here with a sense for the maintenance burden all this entails.

Thank you. If every user understood this like you do, we’d have much more pleasant interactions.

**Pull request closed without merging**